gemrb/core/GUI/GameControl.cpp

```diff
@@ -2249,7 +2282,11 @@ void GameControl::CommandSelectedMovement(const Point& p, unsigned short Mod)
 
 	double angle = 0.0;
```

Comment:
shouldn't `angle` be initialized to `formationBaseAngle` instead? seems like without that non rotation events will always orient to `0.0` instead of in the direction of the party. so now regular clicks and rotations behave differently I think.

Reply:
All regular clicks should go through a call to InitFormation now. In fact, in the previous pull request there was a question about one of these calls.
CommandSelectedMovement can still be called from other paths, such as walking to a door in PerformSelectedAction.